server/game/cards/19.1-AS01/Neutral/MantisRaider.js

```diff
@@ -18,13 +18,13 @@ class MantisRaider extends DrawCard {
         });
 
         this.action({
-            title: 'Give this character +2/+2',
+            title: 'Give this character +1/+0',
```

Comment:
If you're in here editing, you don't need to say it gets +0 pol.

Reply:
Fixed